ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -78,8 +78,10 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
 
         let searchBarNoAutocorrect = buildSearchBar(autocorrectionType: .no, placeholderText: "no autocorrect")
         let searchBarAutocorrect = buildSearchBar(autocorrectionType: .yes, placeholderText: "autocorrect")
+        let numberSearchBar = buildSearchBar(autocorrectionType: .no, placeholderText: "numberpad search")
```

Comment:
nit: can we capitalize n

Reply:
Updated - also capitalized other placeholder texts